src/rpc_client/src/meta_client.rs

```diff
@@ -655,6 +650,10 @@ impl GrpcMetaClient {
         let channel = tokio_retry::Retry::spawn(retry_strategy, || async {
             let endpoint = endpoint.clone();
             endpoint
+                .http2_keep_alive_interval(Duration::from_secs(
```

Comment:
> If the ping is not acknowledged within the timeout, the connection will be closed. Does nothing if http2_keep_alive_interval is disabled.
> 
> Default is 20 seconds.

https://docs.rs/tonic/latest/tonic/transport/struct.Server.html

Reply:
http2_keep_alive_interval is disabled by default in tonic.

Reply:
I did a test to verify the difference http2_keep_alive_interval makes:
- establish a stream, halt the server process (via breakpoint).
- when http2_keep_alive_interval is disabled, client won't see any error. When server is resumed, the stream remain available (can receive message).
- when http2_keep_alive_interval is enabled, client will see error after a while, and the stream is ended (client receive None after previous error. So it need to reestablish another one).